### ISSUE TITLE
Everything seems to work except no images in dest folder?

### ISSUE THREAD

**Issue by v3nt** [gleero/grunt-favicons#46]
They done't seem to be anywhere else on the my machine either.

Settings are 

  favicons: {
    options: {
      trueColor:true,
      precomposed: true,
      appleTouchBackgroundColor: "#e2b2c2",
      coast: true,
      windowsTile: true,
      tileBlackWhite: false,
      tileColor: "auto",
      html: 'templates/head-favicons.php',
      HTMLPrefix: "assest/img/icons/"
    },
    icons: {
      src: 'assets/img/images/Black-Label-Productions.png',
      dest: 'assets/img/icons'
    }

And terminal output is..

Resizing images for "assets/img/images/Black-Label-Productions.png"... sh: line 1:  2772 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 16x16 assets/img/icons/16x16.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-1xsveev.exec 2>&1
sh: line 1:  2775 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 32x32 assets/img/icons/32x32.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-f2eh7q.exec 2>&1
sh: line 1:  2777 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 48x48 assets/img/icons/48x48.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-l6mns0.exec 2>&1
OK
favicon.ico... sh: line 1:  2779 Trace/BPT trap: 5       ( convert assets/img/icons/16x16.png assets/img/icons/32x32.png assets/img/icons/48x48.png -alpha on -background none assets/img/icons/favicon.ico ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-vn7mcb.exec 2>&1
OK
favicon.png... sh: line 1:  2781 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 64x64 assets/img/icons/favicon.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-saw22a.exec 2>&1
OK
apple-touch-icon.png... sh: line 1:  2783 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 57x57 -background "#e2b2c2" -flatten -gravity center -thumbnail "48x48>" -extent 57x57 assets/img/icons/apple-touch-icon.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-1fz852m.exec 2>&1
OK
apple-touch-icon-precomposed.png... sh: line 1:  2785 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 57x57 -background "#e2b2c2" -flatten -gravity center -thumbnail "48x48>" -extent 57x57 assets/img/icons/apple-touch-icon-precomposed.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-1r26diz.exec 2>&1
OK
apple-touch-icon-60x60-precomposed.png... sh: line 1:  2787 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 60x60 -background "#e2b2c2" -flatten -gravity center -thumbnail "51x51>" -extent 60x60 assets/img/icons/apple-touch-icon-60x60-precomposed.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-z74bl0.exec 2>&1
OK
apple-touch-icon-72x72-precomposed.png... sh: line 1:  2789 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 72x72 -background "#e2b2c2" -flatten -gravity center -thumbnail "61x61>" -extent 72x72 assets/img/icons/apple-touch-icon-72x72-precomposed.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-17u63yb.exec 2>&1
OK
apple-touch-icon-76x76-precomposed.png... sh: line 1:  2791 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 76x76 -background "#e2b2c2" -flatten -gravity center -thumbnail "65x65>" -extent 76x76 assets/img/icons/apple-touch-icon-76x76-precomposed.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-11lzard.exec 2>&1
OK
apple-touch-icon-114x114-precomposed.png... sh: line 1:  2793 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 114x114 -background "#e2b2c2" -flatten -gravity center -thumbnail "97x97>" -extent 114x114 assets/img/icons/apple-touch-icon-114x114-precomposed.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-gidiwj.exec 2>&1
OK
apple-touch-icon-120x120-precomposed.png... sh: line 1:  2795 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 120x120 -background "#e2b2c2" -flatten -gravity center -thumbnail "102x102>" -extent 120x120 assets/img/icons/apple-touch-icon-120x120-precomposed.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-1e1w191.exec 2>&1
OK
apple-touch-icon-144x144-precomposed.png... sh: line 1:  2797 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 144x144 -background "#e2b2c2" -flatten -gravity center -thumbnail "122x122>" -extent 144x144 assets/img/icons/apple-touch-icon-144x144-precomposed.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-1guxdu4.exec 2>&1
OK
apple-touch-icon-152x152-precomposed.png... sh: line 1:  2799 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 152x152 -background "#e2b2c2" -flatten -gravity center -thumbnail "129x129>" -extent 152x152 assets/img/icons/apple-touch-icon-152x152-precomposed.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-23r41a.exec 2>&1
OK
coast-icon-228x228.png... sh: line 1:  2801 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 228x228 -background "#e2b2c2" -flatten assets/img/icons/coast-icon-228x228.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-pi0xe1.exec 2>&1
OK
windows-tile-144x144.png... sh: line 1:  2808 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 70x70 assets/img/icons/windows-tile-70x70.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-1n0m6tg.exec 2>&1
sh: line 1:  2810 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 144x144 assets/img/icons/windows-tile-144x144.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-q9mbwl.exec 2>&1
sh: line 1:  2812 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 150x150 assets/img/icons/windows-tile-150x150.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-1mzvg0.exec 2>&1
sh: line 1:  2814 Trace/BPT trap: 5       ( convert assets/img/images/Black-Label-Productions.png -resize 310x310 assets/img/icons/windows-tile-310x310.png ) > /var/folders/73/mmgdtxpx7jnf7xxtcgb65kz00000gp/T/114827-2770-19egsvv.exec 2>&1

**Comment by timodewinter** [comment]
Same here, all works, except for the images...

**Comment by nickspiel** [comment]
Same here... You guys get this working?

**Comment by iparr** [comment]
This just 'magically' started working again. I believe that a problem may have existed with the library that this task uses to resize images, rather than the Grunt task itself. Sorry I can't give more details.

**Comment by nickspiel** [comment]
Yeah just tried it again after a restart, still not working for me. :(

**Comment by v3nt** [comment]
I reinstalled imagemagick and its libraries a few times quite randomly which got it working. Sorry I can't be more specific!
